source/server/guarddog_impl.cc

```diff
@@ -1,6 +1,7 @@
 #include "server/guarddog_impl.h"
 
 #include <chrono>
+#include <iostream>
```

Comment:
this is removed already in a commit I haven't pushed yet because I don't want to waste CI :) Will push when addressing other comments.